DOCKER.md

```diff
@@ -1,36 +1,48 @@
-### Install Docker
+# Install Docker
 
 - [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/)
 
-### Running locally with docker
+_Make sure `docker --version`, `docker-compose --version` and `docker ps` runs without error._
 
-    # Config file.
-    # This file sets which cameras and exposures will be processed
+# Running locally with docker
 
-    cp framework/config/qlf.cfg.template framework/config/qlf.cfg
+### Config file.
+This file sets which cameras and exposures will be processed
 
-    # Test data for local run of QLF: night 20190101, exposures 3 and 4.
+`cp framework/config/qlf.cfg.template framework/config/qlf.cfg`
 
-    mkdir spectro && cd spectro
+### Test data for local run of QLF: night 20190101, exposures 3 and 4.
 
-    wget -c http://portal.nersc.gov/project/desi/data/quicklook/20190101_small.tar.gz 
+```
+mkdir spectro && cd spectro
+wget -c http://portal.nersc.gov/project/desi/data/quicklook/20190101_small.tar.gz 
+```
 
-    # IF using OSX and wget not available, download with curl -O
+_IF using OSX and wget not available, download with curl -O_
 
-    # Unzip
+### Unzip
 
-    tar xvzf 20190101_small.tar.gz
+```
+tar xvzf 20190101_small.tar.gz
+rm 20190101_small.tar.gz
+```
 
-    rm 20190101_small.tar.gz
-
-### Starting QLF
+## Starting QLF
```

Comment:
What do you expect to see?
I would put the Stop information after the Pipeline start.
Also add a note about the size of the container and where it stands (since this bugged Felipe O.)

Reply:
@ricardogando should I also add `reset`?